local.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# Copyright 2021 Angelo Naselli
+# Copyright 2021 SUSE LLC
+# MIT License
+#
+# The default build scripts assume you are using a container or a VM and
+# get their dependencies from /usr and install into /usr.
+# This script sets up a local user prefix on a production machine.
```

Comment:
@anaselli I think it is helpful to include this helper in the main repo so that we can reproduce and smoke-test your setup more easily. Does this match your setup?

Reply:
have you released it into my branch? If so i will do some tests tonight... and see